voltixApp/VoltixApp/View Models/SendCryptoVerifyViewModel.swift

```diff
@@ -164,26 +156,11 @@ class SendCryptoVerifyViewModel: ObservableObject {
                 return keysignPayload
             } else {
                 
-                let estimatedGas = Int64(await estimateGasForERC20Transfer(tx: tx, web3Service: web3Service))
-                
-                guard estimatedGas > 0 else {
-                    errorMessage = "gasEstimateTOKENError"
-                    showAlert = true
-                    isLoading = false
-                    return nil
-                }
-                
-                let decimals: Double = Double(tx.token?.tokenInfo.decimals ?? "18") ?? 18
-                
-                let amountInSmallestUnit: Double = tx.amountDecimal * pow(10.0, decimals)
-                
-                let amountToSend = Int64(amountInSmallestUnit)
-                
-                let keysignPayload = KeysignPayload(
+                                let keysignPayload = KeysignPayload(
                     coin: tx.coin,
                     toAddress: tx.toAddress,
-                    toAmount: amountToSend, // The amount must be in the token decimals
-                    chainSpecific: BlockChainSpecific.ERC20(maxFeePerGasGwei: Int64(tx.gas) ?? 42, priorityFeeGwei: 1, nonce: tx.nonce, gasLimit: Int64(estimatedGas), contractAddr: tx.coin.contractAddress),
+					toAmount: tx.amountInTokenWeiInt64, // The amount must be in the token decimals
+					chainSpecific: BlockChainSpecific.ERC20(maxFeePerGasGwei: Int64(tx.gas) ?? 42, priorityFeeGwei: 1, nonce: tx.nonce, gasLimit: Int64(95000), contractAddr: tx.coin.contractAddress),
```

Comment:
`gasLimit: Int64(95000)` , this is hardcoded ? then we don't need to estimate it above

Reply:
Now it is 120.000 and I removed the methods to estimate them.